### PR TITLE
Add imagestream generator

### PR DESCRIPTION
**Pull request by phracek**
Add support for checking imagestreams before running tests on OpenSHift 4

### CODE REVIEW

**Comment by phracek** [comment]
[test-openshift]

File: imagestreams/imagestreams.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        app_versions: ["5.32"]
+    custom_tags:
+      - name: "5.30"
+        distro: UBI 7
+        app_version: ["5.30"]
+
+  - filename: perl-rhel-aarch64.json
+    latest: "5.32-ubi8"
+    distros:
+      - name: UBI 7
```
Comment (zmiklank) [inline imagestreams/imagestreams.yaml:46]: We do not provide imagestreams for RHEL7 in *-rhel-aarch64.json files.

Comment (phracek) [reply imagestreams/imagestreams.yaml]: Fixed by https://github.com/sclorg/s2i-perl-container/pull/305/commits/ca54731c9b74deedab729f43adb15e5381c12433

**Comment by phracek** [comment]
[test-openshift]

**Pull request merged**